src/components/Facility/PatientNoteCard.tsx

```diff
@@ -44,15 +45,17 @@ const PatientNoteCard = ({
   const [showEditHistory, setShowEditHistory] = useState(false);
   const [editHistory, setEditHistory] = useState<PatientNotesEditModel[]>([]);
   const authUser = useAuthUser();
+  const queryClient = useQueryClient();
 
-  const fetchEditHistory = async () => {
-    const { res, data } = await request(routes.getPatientNoteEditHistory, {
-      pathParams: { patientId, noteId: note.id },
-    });
-    if (res?.status === 200) {
-      setEditHistory(data?.results ?? []);
-    }
-  };
+  const { data, refetch } = useQuery({
+    queryKey: [patientId, note.id],
+    queryFn: async () => {
+      const response = await callApi(routes.getPatientNoteEditHistory, {
+        pathParams: { patientId, noteId: note.id },
+      });
+      return response;
+    },
```

Comment:
stick with how it's in docs. 

avoid calling callApi directly unless absolutely necessary. Stick with `query` or `mutate` wherever needed.

```suggestion
    queryFn: query(routes.getPatientNoteEditHistory, {
      pathParams: { patientId, noteId: note.id },
    },
```